pallets/elections/src/lib.rs

```diff
@@ -59,6 +59,9 @@ pub mod pallet {
     #[pallet::storage]
     pub type MembersPerSession<T> = StorageValue<_, u32, ValueQuery>;
```

Comment:
Technically now we should set it to 0 as default, and set to any given value in e2e test to be aligned with approach for reserved.

Reply:
But the problem is local runs with `run_nodes.sh` would end up in empty committee produced each new session, so this is not that easy I guess. So one would need to run `run_nodes.sh` and set `MembersPerSession` manually via extrinsics, which is not good

Reply:
So we can discuss offline or I am happy for you to resolve this :)

Reply:
I would leave it as it is, since it would make local runs pain for all of us :P